src/nix/add-root.md

```diff
@@ -0,0 +1,18 @@
+R""(
+
+# Examples
+
+  ```console
+  $ readlink foo
+  /nix/store/xxx
+  $ nix add-root foo
```

Comment:
Intuitively I would have expected such as subcommand here:

```suggestion
  $ nix store add-root foo
```

Reply:
Who makes final decisions about that ?

Reply:
Nix team members, I would say. Often happens in those regular meetings.

Reply:
So we discussed this during the meeting:

Our question: Does this command work for you instead of this `nix store add-gc-root`?

It also works with store paths, but also with installables:

```
$ nix build --out-link /tmp/result /nix/store/yb84nwgvixzi9sx9nxssq581pc0cc8p3-hello-2.12.1
$ ls -la /tmp/result
lrwxrwxrwx joerg users 56 B now /tmp/result@ ⇒ /nix/store/yb84nwgvixzi9sx9nxssq581pc0cc8p3-hello-2.12.1
```

The advantage of adding gcroot in general as part of the build is that it race-free.